electron/config.ts

```diff
@@ -240,16 +240,9 @@ abstract class GlobalConfig {
       })
     }
 
-    const steamLibraries = getSteamLibraries()
     const protonPaths = [`${heroicToolsPath}/proton/`]
 
-    // Known places where Steam might be found.
-    // Just add a new string here in case another path is found on another distro.
-    const steamPaths = [...steamLibraries, '/usr/share/steam'].filter((path) =>
-      existsSync(path)
-    )
-
-    steamPaths.forEach((path) => {
+    getSteamLibraries().forEach((path) => {
       protonPaths.push(`${path}/steam/steamapps/common`)
```

Comment:
I think these paths can be updated now. In my case (with two Steam Library folders), I have the following:
- Main folder at `~/.local/share/Steam/`, contains a `compatibilitytools.d` folder for custom Proton and `steamapps/common` for installations
- 2nd one on external drive with just `steamapps/common`

So the first and 3rd path can be dropped

Reply:
But do you remember that the steam folder on ubuntu and derivatives are not the same as Arch? So I think I will keep as it is, for now, to avoid breaking some distro. Its better to be wrong putting too much than too little :P

Reply:
The Steam folder is in another location, that's true. But the library folders point to the same thing (at least they do on my PopOS testing VM, and I'd be *very* surprised if they would store those differently depending on the distro)
But you're right, doesn't really matter